src/mame/drivers/coleco.cpp

```diff
@@ -91,6 +96,40 @@ WRITE8_MEMBER( coleco_state::paddle_on_w )
 	m_joy_mode = 1;
 }
 
+READ8_MEMBER( bit90_state::bankswitch_u4_r )
+{
+	//osd_printf_verbose("Bankswitch to u4\n");
+	m_bank->set_entry(0);
+	return 0xff;
+}
+
+READ8_MEMBER( bit90_state::bankswitch_u3_r )
+{
+	//osd_printf_verbose("Bankswitch to u3\n");
+	m_bank->set_entry(1);
+	return 0xff;
```

Comment:
Three things here:
* You should return `space.unmap()` for the unmapped reads
* You should guard against debugger access in read handlers with side effects by checking `if (!machine().side_effects_disabled())`
* For logging that can be disabled at compile time, you should use the `LOG` macro from `logmacro.h`.